gcampus/core/templates/gcampuscore/components/bach_index.html

```diff
@@ -0,0 +1,114 @@
+{% load i18n measurement %}
+<div class="bg-light me-2">
+    <div class="my-auto p-4">
+
+        <div class="row">
+            <div class="col-9">
+                <div class="text-secondary fw-bold pb-2">
+                    {% translate "BACH Index" %}
+                </div>
+                {% if measurement.bach_index.validity >= 0.25 %}
+                    <div class="index-classification display-1 fw-bolder bach-class-{{ measurement.bach_index.classification }}">
+                        {% translate "Class" %} {{ measurement.bach_index.classification }}
+                    </div>
+                    <div class="index-classification display-4 fw-bolder bach-class-{{ measurement.bach_index.classification }}">
+                        {{ measurement.bach_index.description }}
+                    </div>
+                {% else %}
+                    <div class="index-classification display-1 fw-bolder class-unknown">
+                        {% translate "N/A" %}
```

Comment:
At other places we use a context:

```
{% translate "N/A" context "no value provided (water name, comment, etc)" as not_any %}
```

This will create two translations (one with context and one without). Can you add context and maybe even only translate "N/A" only once (less lookups).